src/app/helptext/sharing/index.ts

```diff
@@ -1,5 +1,11 @@
+import { T } from "app/translate-marker";
+
 export * from "./afp/afp";
 export * from "./iscsi/iscsi";
 export * from "./nfs/nfs";
 export * from "./smb/smb";
 export * from "./webdav/webdav";
+
+export const delete_share_message = (shareName: string) =>
+  T(`Stop sharing. The sharing configuration will be removed.\
```

Comment:
Can this be simplified to
```suggestion
  T(`Stop sharing ${shareName}. The sharing configuration will be removed.\
```

Reply:
How does this interact with the app's translation scheme? I think negatively?

Reply:
The `replace` won't work correctly either for a translated language, so the attempt was futile anyway...

Reply:
Good point.  I don't know whether that will be a problem.  But then, the replacement only works if 'sharing.' is translated the same in both places, and some languages do not necessarily do that (Czech has seven different ways of using nouns, for instance).  Maybe make it more explicit?
```suggestion
  T(`Stop sharing --sharename--. The sharing configuration will be removed.\
```